server/models/Message.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    conversationId: {
+      type: mongoose.ObjectId,
+      ref: "Conversation",
+      required: true,
+    },
+    text: {
+      type: String,
+      required: true,
+    },
+    sender: {
+      type: mongoose.ObjectId,
+      ref: "Profile",
+      required: true,
+    },
+    seen: {
```

Comment:
Though, just my opinion, I'd call this `read`

Reply:
done ✔️